video/image_writer.c

```diff
@@ -440,14 +409,9 @@ static bool write_avif(struct image_writer_ctx *ctx, mp_image_t *image,
         goto free_data;
     }
 
-    if (!overwrite && mp_path_exists(filename)) {
-        MP_ERR(ctx, "File '%s' already exists\n", filename);
-        goto free_data;
-    }
-
-    ret = avio_open(&avioctx, filename, AVIO_FLAG_WRITE);
+    ret = avio_open_dyn_buf(&avioctx);
     if (ret < 0) {
-        MP_ERR(ctx, "Could not open file '%s' for saving images\n", filename);
+        MP_ERR(ctx, "Could not open dyn buf for writing\n");
```

Comment:
could make this `MP_HANDLE_OOM`, it's just allocating a small struct so shouldn't fail